src/PointwiseFunctions/GeneralRelativity/ComputeSpacetimeQuantities.hpp

```diff
@@ -155,7 +155,16 @@ tnsr::A<DataType, SpatialDim, Frame> spacetime_normal_vector(
 /*!
  * \ingroup GeneralRelativityGroup
  * \brief  Computes extrinsic curvature from metric and derivatives.
- * \details Uses ADM evolution equation for spatial metric.
+ * \details Uses the ADM evolution equation for the spatial metric,
+ * \f[ K_{ij} = \frac{1}{2N} \left ( -\partial_0 g_{ij}
+ * + N^k \partial_k g_{ij} + g_{ki} \partial_j N^k
+ * + g_{kj} \partial_i N^k \right ) \f]
+ * where \f$K_{ij}\f$ is the extrinsic curvature, \f$N\f$ is the lapse,
+ * \f$N^i\f$ is the shift, and \f$g_{ij}\f$ is the spatial metric. In terms
+ * of the Lie derivative of the spatial metric with respect to a unit timelike
+ * vector \f$n^i\f$ normal to the spatial slice, this corresponds to the sign
```

Comment:
This is normally denoted as `t^i` in SpEC, no? #1081 I think calls the same thing `t`.

Reply:
In the literature you find `n_a` more often than `t_a`, because `t_a` is sometimes also used for the 1-form `nabla_a t`.
However, the GHG paper uses `t_a`, so we should stick to this notation.

I would also vote for a change of the index to `a` , since it is not a spatial vector.

Reply:
In the literature you find `n_a` more often than `t_a`, because `t_a` is sometimes also used for the 1-form `nabla_a t`.
However, the GHG paper uses `t_a`, so we should stick to this notation.

I would also vote for a change of the index to `a` , since it is not a spatial vector.

Reply:
Ya, I see t_a usually refering to the coordinate timelike vector but I'll go with what we have in SpEC. I'd rather have everything unified. And I'll also switch the index to `a`, I didn't know we had a convention to distinguish them :stuck_out_tongue_closed_eyes:

Reply:
However, everywhere in the spectre documentation so far has `n^a` being the spacetime normal vector. Do you want me to change all of them to `t^a` or do we just want to go with `n^a`?

Reply:
I would be fine with making an issue to change other places in the code